src/hooks/requests/useOrders.ts

```diff
@@ -224,6 +225,28 @@ export const useSubOrderWorkflow = ({ id }: { id?: string }) => {
   }
 }
 
+export const useSplitAssignment = () => {
+  const queryClient = useQueryClient()
+  const { mutateAsync: splitAssignment, isLoading } = useMutation({
+    mutationKey: ['split_assignment'],
+    mutationFn: (payload: {
+      sub_project_id: string
+      job_key: SubProjectFeatures
```

Comment:
[non-blocker]
if you don't use these variables, put the payload type in order type file and import it.